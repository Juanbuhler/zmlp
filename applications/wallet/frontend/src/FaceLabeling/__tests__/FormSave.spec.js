import TestRenderer from 'react-test-renderer'
import FaceLabelingFormSave from '../FormSave'

describe('<FaceLabelingFormSave />', () => {
  it('should render when isLoading', () => {
    const component = TestRenderer.create(
      <FaceLabelingFormSave isChanged isLoading />,
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
