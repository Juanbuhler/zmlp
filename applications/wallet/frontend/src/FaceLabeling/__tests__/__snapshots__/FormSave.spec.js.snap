// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FaceLabelingFormSave /> should render when isLoading 1`] = `"Saving..."`;
